host_core/lib/host_core/control_interface/server.ex

```diff
@@ -198,8 +194,7 @@ defmodule HostCore.ControlInterface.Server do
   # %{"actor_ref" => "bindle://example.com/echo/0.12.0", "host_id" => "Nxxxx", "count" => 4}
   defp handle_request({"cmd", _host_id, "la"}, body, _reply_to) do
     with {:ok, start_actor_command} <- Jason.decode(body),
-         true <-
-           Map.has_key?(start_actor_command, "actor_ref") do
+         actor_ref <- Map.get(start_actor_command, "actor_ref") do
```

Comment:
No matter the contents of the map, this will always succeed even if actor_ref will be nil. Just a note, you don't need it in the `with` statement, but it might be worth checking this with an `is_nil` or something if that's the goal

Reply:
Oops! Good catch.